apps/builder/app/builder/features/seo/site-settings.tsx

```diff
@@ -0,0 +1,210 @@
+  borderColor: theme.colors.borderMain,
+});
+
+const SiteSettingsContent = (props: Props) => {
+  const ids = useIds(["siteName", "favicon", "code"]);
+  const handleChange =
+    <T extends keyof Value>(name: T) =>
+    (val: Value[T]) => {
+      props.onChange({
+        ...props.value,
+        [name]: val,
+      });
+    };
+
+  const assets = useStore(assetsStore);
+  const asset = assets.get(props.value.faviconAssetId ?? "");
+
+  const favIconUrl = asset ? `${asset.name}` : undefined;
+
+  const imageLoader = createImageLoader({
+    imageBaseUrl: env.ASSET_BASE_URL,
+  });
+
+  return (
+    <Grid
+      gap={2}
+      css={{
+        my: theme.spacing[5],
+      }}
+    >
+      <Grid
+        gap={1}
+        css={{
+          width: theme.spacing[34],
+          mx: theme.spacing[5],
+          px: theme.spacing[5],
+        }}
+      >
+        <Label htmlFor={ids.siteName}>Site Name</Label>
+        <InputField
+          id={ids.siteName}
+          value={props.value.siteName ?? ""}
+          onChange={(event) => handleChange("siteName")(event.target.value)}
+          placeholder="Current Site Name"
+          name="Name"
+          autoFocus
+        />
+      </Grid>
+
+      <Separator />
+
+      <Grid gap={2} css={{ mx: theme.spacing[5], px: theme.spacing[5] }}>
+        <Label htmlFor={ids.favicon} sectionTitle>
+          Favicon
+        </Label>
+        <Grid flow="column" gap={3}>
+          <Image
+            width={72}
+            height={72}
+            className={imgStyle()}
+            src={favIconUrl}
+            loader={imageLoader}
+          />
+
+          <Grid gap={2}>
+            <Text color="subtle">
+              Upload a 32 x 32 px image to display in browser tabs.
+            </Text>
+            <ImageControl
+              assetId={props.value.faviconAssetId ?? ""}
+              onAssetIdChange={handleChange("faviconAssetId")}
+            >
+              <Button id={ids.favicon} css={{ justifySelf: "start" }}>
+                Upload
+              </Button>
+            </ImageControl>
+          </Grid>
+        </Grid>
+      </Grid>
+
+      <Separator />
+
+      <Grid gap={2} css={{ mx: theme.spacing[5], px: theme.spacing[5] }}>
+        <Label htmlFor={ids.code} sectionTitle>
+          Custom Code
+        </Label>
+        <Text color="subtle">
+          Custom code and scripts will be added at the end of the &lt;head&gt;
+          tag to every page across the published site.
+        </Text>
+        <TextArea
+          id={ids.code}
+          rows={5}
+          autoGrow
+          maxRows={10}
+          value={props.value.code ?? ""}
+          onChange={handleChange("code")}
+        />
+      </Grid>
+      <div />
+    </Grid>
+  );
+};
+
+const SiteSettingsView = () => {
+  const [value, setValue] = useState(
+    pagesStore.get()?.meta ?? {
+      siteName: "",
+      faviconAssetId: "",
+      code: "",
+    }
+  );
+
+  const open = useStore($isSiteSettigsOpen);
```

Comment:
would be nice to use isSomething for booleans (I know radix doesn't follow that) 
```suggestion
  const isOpen = useStore($isSiteSettigsOpen);
```